### ISSUE TITLE
Image: v1.21.0-k3s1 fails when running in a container

### ISSUE THREAD

**Issue by iwilltry42** [k3s-io/k3s#3296]
**Environmental Info:**
K3s Version: 

k3s version v1.21.0+k3s1 (2705431d)
go version go1.16.2

via k3d v4.4.3

Node(s) CPU architecture, OS, and Version: 

OpenSUSE Tumbleweed

Cluster Configuration:

Single-Node cluster using `k3d cluster create`

**Describe the bug:**

Containerd fails to create containers
```
failed to create containerd task: OCI runtime create failed: container_linux.go:380:
          starting container process caused: apply caps: operation not permitted:
          unknown
```

**References**

- First catched by @ibuildthecloud 
- Similar issue in `kind`: https://github.com/kubernetes-sigs/kind/issues/2058
  - underlying problem fixed in containerd by @AkihiroSuda: https://github.com/containerd/containerd/pull/5017 (released in v1.5.0-beta.2)
  - fixed in `kind` by updating the deps: https://github.com/kubernetes-sigs/kind/pull/2084

I guess this would require an update of https://github.com/k3s-io/containerd in the end :thinking: 
@AkihiroSuda do you have some more input on this issue maybe?

**Comment by davidnuzik** [comment]
Assigned to @iwilltry42 for checking if this is resolved in k3s 1.21.1 - if so please close this issue.

**Comment by iwilltry42** [comment]
@davidnuzik , I just gave it a try on that same test system without any issues :+1: 
Thanks for the fix guys!
